### PR TITLE
Policy details page

### PR DESCRIPTION
**Pull request by forevermatt**
### Added
- Add formatFriendlyDate() helper
- Add basic policy-details page, with links to claim details

### Fixed
- Prevent error from trying to get properties of an item that wasn't found

### CODE REVIEW

**Review by RussellChamp — APPROVED** [review]
lgtm

File: src/helpers/date.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+export const formatFriendlyDate = (dateTimeString: string) => {
+  const date = new Date(dateTimeString)
+  return date.toLocaleDateString()
+}
```
Comment (hobbitronics) [inline src/helpers/date.ts:5]: we have a /components/dates.js file that does basically this with a few parameters. Could you see if that meets the requirement and/or move both functions to the same file.

**Review by hobbitronics — COMMENTED** [review]
Looks good.

File: src/helpers/date.ts (hunk 1 above)
Comment (forevermatt) [reply src/helpers/date.ts:5]: Ahh, thanks! I thought we had one, but I wasn't seeing it. I'll take a look at combining those.

**Pull request merged**